build.gradle.kts

```diff
@@ -26,7 +26,7 @@ configurations.all {
         failOnVersionConflict()
         eachDependency {
             when (requested.module.toString()) {
-                "com.google.guava:guava" -> useVersion("23.6-jre")
+                "com.google.guava:guava" -> useVersion("25.0-jre") // conflict between infrastructure, virtual-users and jira-actions
```

Comment:
Commit says "Remove direct Guava dependency", but the dependency is still here... Am I missing something?

Reply:
This is an override of transitive conflict. The direct dependency is deleted below in the `dependencies` section

Reply:
Oh, right. I missed it.